views/mail/customer/created.htm

```diff
@@ -1,18 +1,18 @@
-subject = "Welcome to our store, {{ user.customer.firstname }}!"
+subject = "{{ 'offline.mall::mail.customer.created.subject' | trans() }} {{ user.customer.firstname }}!"
 description = "Sent after a customer signs up"
 ==
 
-Hi {{ user.customer.firstname }}
+{{ 'offline.mall::mail.common.hello' | trans() }} {{ order.customer.firstname }}
 
 {% if confirm %}
-    Welcome to our store! Please click on the button below to confirm your e-mail address.
+    {{ 'offline.mall::mail.customer.created.confirm_mail' | trans() }}
 {% else %}
-    Welcome to our store! You can log in using your e-mail address {{ user.email }}
-    and start shopping immediately.
+    {{ 'offline.mall::mail.customer.created.created_message.0' | trans() }} {{ user.email }}
```

Comment:
Instead of splitting these lines up, try to use placeholders in the translation strings:
https://laravel.com/docs/5.5/localization#replacing-parameters-in-translation-strings

```twig
{# You can log in using your e-mail address :email to our store #}
{{ 'offline.mall::mail.customer.created.created_message' | trans({email: user.email}) }}
```